.organizationmap

```diff
@@ -391,6 +391,7 @@ The Linux Box <contact@linuxbox.com> Adam C. Emerson <aemerson@linuxbox.com>
 The Linux Box <contact@linuxbox.com> Ali Maredia <ali@linuxbox.com>
 The Linux Box <contact@linuxbox.com> Casey Bodley <casey@linuxbox.com>
 The Linux Box <contact@linuxbox.com> Matt Benjamin <matt@linuxbox.com>
+The Linux Foundation <info@linuxfoundation.org> Shinobu Kinjo <shinobu@linux.com>
```

Comment:
Le 26/08/16 à 18:50, Kefu Chai a écrit :
>
> In .organizationmap 
> <https://github.com/ceph/ceph/pull/9301#discussion_r76450554>:
>
> > @@ -391,6 +391,7 @@ The Linux Box <contact@linuxbox.com> Adam C. Emerson <aemerson@linuxbox.com>
> >  The Linux Box <contact@linuxbox.com> Ali Maredia <ali@linuxbox.com>
> >  The Linux Box <contact@linuxbox.com> Casey Bodley <casey@linuxbox.com>
> >  The Linux Box <contact@linuxbox.com> Matt Benjamin <matt@linuxbox.com>
> > +The Linux Foundation <info@linuxfoundation.org> Shinobu Kinjo <shinobu@linux.com>
>
> i think shinobu is from redhat.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/ceph/ceph/pull/9301/files/4b89878424823ebb921c802fba377a032151f44c..d405ad4296ead7a57d4b1da1639ba8f1263ed74b#r76450554>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AGvyZtitJCj_Rr73KcEkGb2A4iPoyNg1ks5qjxlngaJpZM4Il6b3>.
>
Hi, I sent Shinobu some mails at the start of June, regarding this entry 
because I had some doubts how to properly normalize his mail.

Now, taking your mail into consideration, re-reading his answers & 
thinking twice, I may have misunderstood the sense of Shinobu's last 
answer, and done the exact opposite of what he wanted.
Will Recontact shinobu right now.

Cheers,